src/EVM/Op.hs

```diff
@@ -335,6 +337,8 @@ getOp x = case x of
   0x59 -> OpMsize
   0x5a -> OpGas
   0x5b -> OpJumpdest
+  0x5c -> OpTload
```

Comment:
I think it may be best calling it `TStore` and `TLoad` instead of `Tstore` and `Tload`